### DISCUSSION TITLE
The translation source of the store descrition should be updated

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#1857] [General]
The current translation source not only differs from the English text of the store, but also contains many outdated descriptions. It listed many dead filterlists, and claims that "It's quite an early version, keep this in mind when you review.", even though uBlock Origin is really stable.

According to https://github.com/uBlockOrigin/uBlock-issues/issues/971#issuecomment-657038195,
> Crowdin is terrible for plain text file, modifying one part invalidate everything.

but the English description has been completely rewritten compared to the translation source, so I don't see any problem in redoing the translation from scratch.

**Comment by ghost** [comment]
@gorhill maybe split the text into paragraphs and put into JSON? [more or less where `***` lines are now] Translating paragraphs should be easier than sentence-by sentence.

**Reply by gorhill** [reply]
Yes I could do something like that, with the name of each paragraph being something like `"p1"` and incrementing the number for each new paragraph. What is blocking me is that I don't think I am the best to write descriptions, I feel it's something done by the community through consensus.

**Reply by ghost** [reply]
Maybe announce this on Crowdin and link to: https://github.com/uBlockOrigin/uBlock-issues/wiki/Reference-description-of-uBO-in-various-extensions-stores

Maybe Crowdin has more people interested about language than GitHub, where are more coders?

**Reply by comradekingu** [reply]
I suggested using Weblate before, and it remains an option that isn't broken.

**Comment by ghost** [comment]
Another feedback & discussion: https://old.reddit.com/r/uBlockOrigin/comments/rmtuip/outdated_description_of_the_extension/

**Comment by ghost** [comment]
Mozilla announced their new API for submitting and updating add-ons.
https://blog.mozilla.org/addons/2022/03/17/new-api-for-submitting-and-updating-add-ons/

Possible to improve the workflow?
